config/locales/en.yml

```diff
@@ -48,3 +52,10 @@ en:
       invoice_slip_legend: Invoice/Slip Content
       invoice_number: 'Invoice number: %{number}'
       invoice_date: 'Invoice date: %{date}'
+      date: Date
+      number: Number
+      firstname: First name
+      lastname: Last name
+      document_type: Document type
+      email: Email
+      total: Total
```

Comment:
Please add empty line at the end of file.
Also in config/locales/zh-TW.yml

Reply:
What about `app/views/spree/admin/orders/_print_invoice_order_tab_links.html.erb`
I think this file also should have small changes 
```
<% if can? :show, Spree::Order %>
  <% if @order && @order.completed_at? %>
      <li data-hook="admin_order_tabs_documents">
        <%=
          link_to_with_icon 'file',
            Spree.t(:documents, scope: :print_invoice),
            spree.admin_order_bookkeeping_documents_path(@order),
            title: title,
            class: "#{'active ' if current == :documents}nav-link icon-link with-tip action-refresh"
        %>
      </li>
  <% end %>
<% end %>
```